fdbserver/Knobs.cpp

```diff
@@ -657,6 +657,12 @@ void ServerKnobs::initialize(bool randomize, ClientKnobs* clientKnobs, bool isSi
 
 	// clang-format on
 
-	if(clientKnobs)
-		clientKnobs->IS_ACCEPTABLE_DELAY = clientKnobs->IS_ACCEPTABLE_DELAY*std::min(MAX_READ_TRANSACTION_LIFE_VERSIONS, MAX_WRITE_TRANSACTION_LIFE_VERSIONS)/(5.0*VERSIONS_PER_SECOND);
+	if(clientKnobs) {
+		clientKnobs->IS_ACCEPTABLE_DELAY =
+		    clientKnobs->IS_ACCEPTABLE_DELAY *
+		    std::min(MAX_READ_TRANSACTION_LIFE_VERSIONS, MAX_WRITE_TRANSACTION_LIFE_VERSIONS) /
+		    (5.0 * VERSIONS_PER_SECOND);
+		clientKnobs->OPERATION_COST_BYTE_FACTOR = OPERATION_COST_BYTE_FACTOR;
```

Comment:
I just want to keep the server and client have the same value of OPERATION_COST_BYTE_FACTOR. Because both ss and the client use this to calculate the cost. I think this value should depend on the server, considering the page size of SS is transparent to the client. But the client cannot get the value in SERVER_KNOB. I omit the fact CLIENT_KNOB can be overridden here, though.
Is there better ways to force them the same?

Reply:
I think this can just be a client knob, because the server has access to client knobs. The client is already relying on the page size of storage servers for its cost calculations. So we can remove `ServerKnobs::OPERATION_COST_BYTE_FACTOR`.

Reply:
I think I would be in favor of keeping two separate knobs, which gives us the flexibility to modify these independently. I don't think there's any particular relation between the two knobs that would require them to be the same, and my expectation would be that they'd actually be different.